scaleway/resource_lb_backend_beta.go

```diff
@@ -0,0 +1,301 @@
+			"lb_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The load-balancer id",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the backend",
+			},
+			"forward_protocol": {
+				Type: schema.TypeString,
+				ValidateFunc: validation.StringInSlice([]string{
+					lb.ProtocolTCP.String(),
+					lb.ProtocolHTTP.String(),
+				}, false),
+				Required:    true,
+				Description: "Backend protocol",
+			},
+			"forward_port": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "User sessions will be forwarded to this port of backend servers",
+			},
+			"forward_port_algorithm": {
+				Type: schema.TypeString,
+				ValidateFunc: validation.StringInSlice([]string{
+					lb.ForwardPortAlgorithmRoundrobin.String(),
+					lb.ForwardPortAlgorithmLeastconn.String(),
+				}, false),
+				Default:     lb.ForwardPortAlgorithmRoundrobin.String(),
+				Optional:    true,
+				Description: "Load balancing algorithm",
+			},
+			"sticky_sessions": {
+				Type: schema.TypeString,
+				ValidateFunc: validation.StringInSlice([]string{
+					"none",
+					lb.StickySessionsTypeCookie.String(),
+					lb.StickySessionsTypeTable.String(),
+				}, false),
+				Optional:    true,
+				Description: "Load balancing algorithm",
+			},
+			"sticky_sessions_cookie_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Cookie name for for sticky sessions",
+			},
+			"server_ips": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Description: "Backend server IP addresses list (IPv4 or IPv6)",
+			},
+			"send_proxy_v2": {
+				Type:        schema.TypeBool,
+				Description: "Enables PROXY protocol version 2",
+				Optional:    true,
+				Default:     false,
+			},
+			"timeout_server": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Maximum server connection inactivity time (in milliseconds).",
+			},
+			"timeout_connect": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Maximum initial server connection establishment time (in milliseconds).",
+			},
+			"timeout_tunnel": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Maximum tunnel inactivity time (in milliseconds).",
+			},
+			"on_marked_down_action": {
+				Type: schema.TypeString,
+				ValidateFunc: validation.StringInSlice([]string{
+					"none",
+					lb.OnMarkedDownActionShutdownSessions.String(),
+				}, false),
+				Optional:    true,
+				Description: "Modify what occurs when a backend server is marked down.",
+			},
+		},
+	}
+}
+
+func resourceScalewayLbBackendBetaCreate(d *schema.ResourceData, m interface{}) error {
+	lbAPI := getLbAPI(m)
```

Comment:
nit: geLbAPIWithRegion(d, m)

Reply:
no because we want the the region of `lb_id` not `id`